lib/mwUtil.js

```diff
@@ -238,9 +238,24 @@ mwUtil.getQueryString = function(req) {
     return '';
 };
 
-mwUtil.createRelativeTitleRedirect = function(path, req, newReqParams, titleParamName) {
-    titleParamName = titleParamName || 'title';
-    newReqParams = newReqParams || req.params;
+/**
+ * Create a `location` header value for a relative redirect.
+ *
+ * @param {string} path, the path pattern from specInfo.
+ * @param {hyper.request} req, the request
+ * @param {object} options, with possible parameters:
+ *    - {object} newReqParams, use these parameters instead of the original
+ *    request parameters.
+ *    - {string} titleParamName, the name of the title parameter.
+ *    - {boolean} dropPathAfterTitle, indicating that the redirect should drop
+ *    the path after the title parameter. Typically used for redirects to
+ *    another title, where revision & tid would no longer match.
+ * @return {string} Location header value containing a relative redirect path.
+ */
```

Comment:
Good to know, thanks for the pointer! Changed the syntax to use the dotted-name syntax.